catalyst-gateway/bin/src/db/event/signed_docs/sql/select_signed_documents.sql.jinja

```diff
@@ -0,0 +1,12 @@
+SELECT
+  signed_docs.type,
+  signed_docs.author,
+  signed_docs.metadata,
+  signed_docs.payload,
+  signed_docs.raw
+FROM signed_docs
+WHERE
+  signed_docs.id = '{{ id }}'
+  AND signed_docs.ver = '{{ ver }}'
```

Comment:
Should be optional. so that if `ver` is not set we just get the latest `ver` (because of the ORDER BY).

Reply:
This query is for private use and used only with the `insert_signed_documents` query. So I can check that the document exists inside the db or not. That is why I've made it that simple, because it's just for one use case.

Do we want to make it public ? Because from what we discussed, we are gonna use only `filtered_select_...` query.

Reply:
No, as I said, filtered_select is used to find documents by their metadata.  We need a function to retrieve a single document.